app/src/main/java/com/android/solvit/shared/ui/chat/ChatScreen.kt

```diff
@@ -69,13 +72,19 @@ fun ChatScreen(
   val receiverName = getReceiverName(receiver)
   val receiverPicture = getReceiverImageUrl(receiver)
 
+  val user by authViewModel.user.collectAsState()
+  chatAssistantViewModel.setContext(messages, "Hassan", receiverName)
```

Comment:
Is it normal that the sender name is hard coded ??

Reply:
It is not, but it is hardcoded also in the text messages, so I could not make better!

Reply:
Yes Indeed! I'll make a PR to correct it !